dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityOperationParamsMapper.java

```diff
@@ -413,11 +413,11 @@ private boolean isProgramMinAttributesViolated(TrackedEntityQueryParams params)
                 < params.getEnrolledInTrackerProgram().getMinAttributesRequiredToSearch());
   }
 
-  private void checkIfMaxTeiLimitIsReached(TrackedEntityQueryParams params, int maxTeiLimit) {
-    if (maxTeiLimit > 0) {
+  private void checkIfMaxTeLimitIsReached(TrackedEntityQueryParams params, int maxTeLimit) {
+    if (maxTeLimit > 0) {
```

Comment:
NitPicky, but if the `maxTeLimit` is not sent as an additional parameter, and if reusing the one inside `params` , a method like `hasMaxTeLimit` inside `TrackedEntityQueryParams` could be used here instead, that does the same check.

Reply:
I should not have renamed the existing code 😂